packages/realm-server/tests/loader-test.ts

```diff
@@ -85,22 +85,6 @@ module('loader', function (hooks) {
     ]);
   });
 
-  test('can determine consumed modules when an error is encountered during loading', async function (assert) {
```

Comment:
this test was not super useful, and now that we are caching modules we can't see this kind of detail behind the cache. but that should be fine as we were never relying on this behavior